### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by H4wking**

### CODE REVIEW

**Comment by pavlo-berezin** [comment]
image
Cannot start.

**Comment by H4wking** [comment]
Пофіксив, створив пустий ксс на майбутнє і забув що пусті файли видаляються.

File: src/components/transactions.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                    date: "03.01.20",
+                    category: "Car",
+                    amount: 350,
+                }
+            ]
+        };
+    }
+
+    calcInflow(transactions) {
+        var inflow = 0;
```
Comment (pavlo-berezin) [inline src/components/transactions.js:39]: you can use `reduce` here.

File: src/components/transactions.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        for (var transaction of transactions) {
+            if (!transaction.spent) {
+                inflow += transaction.amount
+            }
+        }
+        return inflow
+    }
+
+    calcOutflow(transactions) {
+        var inflow = 0;
```
Comment (pavlo-berezin) [inline src/components/transactions.js:49]: `outflow`

File: src/components/transactions.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        for (var transaction of transactions) {
+            if (transaction.spent) {
+                inflow += transaction.amount
+            }
+        }
+        return inflow
+    }
+
+    openForm() {
+        document.getElementById("add-transaction").style.display = "block";
```
Comment (pavlo-berezin) [inline src/components/transactions.js:59]: this is bad. Use dynamic styles that react present.

File: src/components/transactions.js (hunk 1 above)
Comment (H4wking) [reply src/components/transactions.js:39]: Will fix in the next homework.

File: src/components/transactions.js (hunk 2 above)
Comment (H4wking) [reply src/components/transactions.js:49]: Will fix in the next homework.

File: src/components/transactions.js (hunk 3 above)
Comment (H4wking) [reply src/components/transactions.js:59]: Removed it in the next homework, changed the way form works.

**Pull request merged**